extensions/PhabBugz/Extension.pm

```diff
@@ -18,6 +18,26 @@ use Bugzilla::Extension::PhabBugz::Feed;
 
 our $VERSION = '0.01';
 
+sub mojo_after_dispatch {
+    my ($c) = @_;
+    my $stash = $c->stash;
+
+    return unless $stash->{controller} eq 'CGI' && $stash->{action} eq 'rest_cgi';
+    return unless $stash->{PATH_INFO} =~ m{^phabbugz/build_target};
+    my $datadog = Bugzilla->datadog('bugzilla.phabbugz') or return;
```

Comment:
Do we want to place all API requests/errors/duration monitoring under a `bugzilla.api.*` namespace?